config/ftbquests/quests/chapters/important.snbt

```diff
@@ -142,13 +142,13 @@
 		{
 			x: 0.0d
 			y: 0.5d
-			description: ["Some loot boxes can be crafted to obtain certain loot. Other loot boxes can be collected via quest rewards and the amadron shop"]
+			description: ["一些战利品可以通过合成战利品箱来获得。战利品箱也可以通过收集任务奖励以及在亚马龙处购买"]
 			dependencies: ["42930831093A12C6"]
 			id: "0D8EA6350C211C2F"
 			tasks: [{
 				id: "17B5C95BCDEDCE98"
 				type: "checkmark"
-				title: "Loot Boxes"
+				title: "战利品箱"
```

Comment:
```suggestion
				title: "战利品盒"
```

https://github.com/ShaBaiTianCN/Minecraft-Mod-Language-Package-Not-Included-Language-Package/pull/16/files#diff-e4f94d02d6561b2e382f6cb090b024dd8ce87e6f8baafa62189dc028ca6cf6d5R22-R26